examples/src/examples/user-interface/button-basic.tsx

```diff
@@ -79,9 +79,7 @@ class ButtonBasicExample {
 
                 // Button
                 const button = new pc.Entity();
-                button.addComponent("button", {
-                    imageEntity: button
-                });
+                button.addComponent("button", {});
```

Comment:
```suggestion
                button.addComponent("button");
```

Reply:
And is the `button` entity referenced anywhere else?

Reply:
In the onclick events.